src/lib/coerce.js

```diff
@@ -114,8 +114,13 @@ exports.valObjectMeta = {
             'are coerced to the `dflt`.'
         ].join(' '),
         requiredOpts: [],
-        otherOpts: ['dflt', 'min', 'max', 'arrayOk'],
+        otherOpts: ['dflt', 'min', 'max', 'arrayOk', 'extras'],
```

Comment:
@archmoj Is this adding a new functionality to `coerce`, where numeric (or other) properties can also be configured to accept a list of 'extra' values?

That's pretty cool -- is there anywhere we should document that internally?

Reply:
Good question.
Yes that's true.
We have other types that support the `extras` option so I think it's clear internally.
But on plotly.py we need to see if this changes would be reflected properly by codegen.

Reply:
Good point.

Makes me think, it would be awesome if we could build checking the Plotly.py codegen into the CI process. It would have to produce some kind of output that's easy to verify. It could run only if the `schema.json` has changed.

Reply:
Perhaps add a line in the docstring here explaining `extras` for integers (like the explanation [here](https://github.com/plotly/plotly.js/blob/353efc4090727bb486120ad6ebca90679d673a3a/src/lib/coerce.js#L243) for `flaglist`)

Reply:
For flag lists combination mean something. That's why it is commented.